### PR TITLE
Mateus/manifest v3

### PR DESCRIPTION
**Pull request by samuraiexx**
- Change Chrome Extension Manifest version to v3 based on https://developer.chrome.com/docs/extensions/develop/migrate
- Remove intro skip feature (Crunchyroll now supports that natively)

### CODE REVIEW

File: manifest.json [hunk 1]
```diff
@@ -1,50 +1,40 @@
...
+  },
+  "icons": {
+    "16": "images/get_started16.png",
+    "32": "images/get_started32.png",
+    "48": "images/get_started48.png",
+    "128": "images/get_started128.png"
+  },
+  "options_page": "options.html",
+  "manifest_version": 3,
+  "minimum_chrome_version": "116"
```
Comment (hsnavarro) [inline manifest.json:39]: Just curious, where did you get this 'minimum_chrome_version' value from?

File: src/options.ts [hunk 2]
```diff
@@ -102,7 +84,7 @@ addButton.onclick = function (): void {
     setColorMenu(colorOptions);
     updateColorMenu(colorOptions);
   });
-}
+};
```
Comment (hsnavarro) [inline src/options.ts:87]: NIT: is this ';' necessary?

File: manifest.json (hunk 1 above)
Comment (samuraiexx) [reply manifest.json:39]: From here: [Chrome 116 introduced the following service worker lifetime improvements](https://developer.chrome.com/docs/extensions/develop/concepts/service-workers/lifecycle#:~:text=Chrome%20116%20introduced%20the%20following%20service%20worker%20lifetime%20improvements%3A)

Basically it allows websocket messages to extend the lifetime of the extension.

File: src/options.ts (hunk 2 above)
Comment (samuraiexx) [reply src/options.ts:87]: It is not, let me remove.

**Pull request merged**